firestore/cloud-client/vector_search.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+
+from google.cloud import firestore
+
+
+def store_vectors():
+    # [START firestore_store_vectors]
+    from google.cloud import firestore
+    from google.cloud.firestore_v1.vector import Vector
+
+    firestore_client = firestore.Client()
+    collection = firestore_client.collection("coffee-beans")
+    doc = {
+        "name": "Kahawa coffee beans",
+        "description": "Information about the Kahawa coffee beans.",
+        "embedding_field": Vector([1.0, 2.0, 3.0]),
+    }
+
+    collection.add(doc)
+    # [END firestore_store_vectors]
+
+
+def vector_search_basic():
+    db = firestore.Client(
+        add_unique_string=False
+    )  # Flag for testing purposes, needs index to be precreated
```

Comment:
nit: it looks like the tests also create a client instance. You could make the functions here take in that client as an argument to avoid the extra instantiation

Reply:
Done